extensions/default/src/commandsModule.ts

```diff
@@ -429,7 +423,14 @@ const commandsModule = ({
           );
 
           return viewport
-            ? { viewportOptions, displaySetOptions, ...viewport }
+            ? {
+                displaySetOptions,
+                ...viewport,
+                viewportOptions: {
+                  ...viewportOptions,
```

Comment:
Why the double spread here? `viewportOptions` I think is that of the viewport that was in one-up and was double clicked. I am not sure why those `viewportOptions` would apply to EVERY arbitrary viewport in the grid? Perhaps I am missing something.